lib/resources/crontab.rb

```diff
@@ -0,0 +1,81 @@
+# encoding: utf-8
+# author: Adam Leff
+
+require 'utils/parser'
+require 'utils/filter'
+
+module Inspec::Resources
+  class Crontab < Inspec.resource(1)
+    name 'crontab'
+    desc 'Use the crontab InSpec audit resource to test the contents of the crontab for a given user which contains information about scheduled tasks owned by that user.'
+    example "
+      describe crontab('root') do
+        its('commands') { should include '/path/to/some/script' }
+      end
+
+      describe crontab('myuser').command('/home/myuser/build.sh') do
```

Comment:
Not sure that the resource supports the singular right now. It works with `commands` but not `command` out of the box. That's a whole other discussion, but lovely to solve :)